examples/apps/color_finder.py

```diff
@@ -673,9 +674,15 @@ def get_blob_center(self):
         return blob_center
 
     def update_largest_blob_size(self, key):
+        '''Updates self.largest_blob_size based on the current largest blob.'''
```

Comment:
that isn't strictly true - the method will set largest_blob_size to the size of the key blob. In this case, it might be best to just remove the method (as it's just 1 line), and change the calling code to:
```python
self.largest_blob_size = len(self.blobs_dict[largest_blob_key])
```